docs/source/storage/general_storage.rst

```diff
@@ -41,27 +39,7 @@ You can schedule an online consultation on the `RC website <https://rc.northeast
    If you are not connected to the campus internet, you must be connected to
    the university's VPN (GlobalProtect) before you can access these storage systems.
    You can find detailed information about downloading and using the GlobalProtect VPN
-   in the `ServiceNow Knowledge Base <https://service.northeastern.edu/tech?id=kb_article&sys_id=4701e07adb93485084ba5595ce9619a9>`_.
-
-Connecting to /research
-========================
-After you have received notification that your shared storage folder on /research is ready for you to use, you
-can connect to it on your laptop. Use the following procedures to connect to /research from either a Windows PC or a Macbook.
-If you are not on the campus internet, you will need to first be connected to Northeastern's VPN before you can connect to /research.
-
-**Windows**
+   in the `ServiceNow Knowledge Base <https://service.northeastern.edu/tech?id=kb_article_view&sysparm_article=KB0013983>`_.
 
- 1. Open **File Explorer**.
- 2. In the left pane, select **This PC**.
- 3. On the **Computer** tab, select **Map network drive**.
- 4. Select any letter not currently mapped to a drive. It does not matter what letter.
- 5. In the **Folder** box, type ``\\nunet.neu.edu\rc-shares\<sharename>``, where ``<sharename>`` is the name of your shared storage. If you want this drive to reconnect automaically every time you log in, check Reconnect at sign in.
- 6. Click Finish. Your shared folder now be available in File Explorer.
-
-**Mac**
-
- 1. Click the **Finder** icon, and in the **Finder** window, select **Go**, then click **Connect to Server**.
- 2. In the **Server Address** field type ``smb://nunet.neu.edu/rc-shares/<sharename>`` where ``<sharename>`` is the name of the share you want to connect to.
- 3. Click Connect.
- 4. If you are prompted to enter your name and password for the nunet server, select **Connect as Registered User**, in the **Name** field type ``NUNET\<yourusername>``, (where ``<yourusername>`` is your NU username) and type your NU password in the **Password** field. If you want to connect to /research automatically when you log in to your computer, select **Remember this password in my keychain**.
- 5. Click Connect. Your share should appear as a folder in the Finder window.
+.. note::
+   The /research storage tier is no longer provided. If you are a former user of /research and have questions or issues related to /research, please `submit a ticket to RC <https://bit.ly/NURC-Assistance>`_. Other storage options include /work (more information found here, :ref:`discovery_storage`), `Sharepoint <https://service.northeastern.edu/tech?id=kb_article&sysparm_article=KB0012695>`_, and `OneDrive <https://service.northeastern.edu/tech?id=kb_article&sysparm_article=KB0012764>`_. 
```

Comment:
I haven't made this change yet because if we do combine the pages, the link would point back to the same page. So I am holding off until we made a decision on that.